### PR TITLE
Trying to address https://github.com/home-assistant/home-assistant.io…

### PR DESCRIPTION
**Pull request by nickrout**
…/issues/15721

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/installation.markdown [hunk 1]
```diff
@@ -46,7 +46,7 @@ These install options are fully supported by Home Assistant's documentation. For
 
 <div class='note'>
 
-The only installation methods that allow you to use Home Assistant Add-ons is using the Home Assistant image. All other methods only install the base Home Assistant packages, however the software from the add-ons may still usually be installed manually like any other program.
+The only installation methods that allow you to use Home Assistant Add-ons is using the [Home Assistant OS](/hassio/installation/)|[A supported platform](/hassio/installation/) image or a [Home Assistant Supervised](https://github.com/home-assistant/supervised-installer) install. All other methods only install the base Home Assistant packages, however the software from the add-ons may still usually be installed manually like any other program.
```
Comment (frenck) [inline source/_docs/installation.markdown:49]: The supported installation images linked are also all Home Assistant OS, thus you are listing a duplicate?

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by nickrout** [comment]
Better now?

**Comment by frenck** [comment]
@nickrout I don't see any changes? Forgot to push up your changes?

**Comment by frenck** [comment]
@nickrout ?

**Comment by nickrout** [comment]
Sorry work has been busy. Long weekend this weekend, I'll have some time
hopefully.

**Comment by frenck** [comment]
The installation pages have been fully re-designed. The change in this PR no longer applies.

**Pull request closed without merging**